src/test/java/org/jenkinsci/plugins/workflow/steps/scm/SCMStepTest.java

```diff
@@ -32,10 +32,12 @@
 import java.io.File;
 import java.util.Iterator;
 import java.util.List;
-import jenkins.plugins.git.GitSampleRepoRule;
+
+import jenkins.plugins.git.*;
```

Comment:
BTW general code style is to avoid wildcard imports except for `static` imports of standard test utilities.